templates/gitlab/myrror-cli.gitlab-ci.yml

```diff
@@ -4,8 +4,34 @@ myrror-scan:
   before_script:
     - MYRROR_REPOSITORY=$CI_PROJECT_PATH
     - MYRROR_ROOT_NAMESPACE=$CI_PROJECT_ROOT_NAMESPACE
+    - apt-get -qq update
+    - apt-get install -y jq
+    - |
+      if [ "$CI_PIPELINE_SOURCE" == "merge_request_event" ]; then
+        MR_DETAILS=$(curl --header "PRIVATE-TOKEN: $ACCESS_TOKEN" "https://gitlab.com/api/v4/projects/$CI_PROJECT_ID/merge_requests/$CI_MERGE_REQUEST_IID")
+        MYRROR_MERGE_REQUEST_STATE=$(echo "$MR_DETAILS" | jq -r ".state")
+        MYRROR_MERGE_REQUEST_CREATED_AT=$(echo "$MR_DETAILS" | jq -r ".created_at")
+        MYRROR_MERGE_REQUEST_UPDATED_AT=$(echo "$MR_DETAILS" | jq -r ".updated_at")
```

Comment:
these fields does not exist in [Predefined CI/CD variables](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html)